openhtf/util/xmlrpcutil.py

```diff
@@ -16,13 +16,20 @@
 
 import http.client
 import xmlrpc.server
+import os
 import socketserver
+import sys
 import threading
 import xmlrpc.client
 import collections
 
 DEFAULT_PROXY_TIMEOUT_S = 3
 
+if sys.version_info[0] < 3:
```

Comment:
http://python-future.org/compatible_idioms.html#xmlrpc-module makes it look like we shouldn't need to switch on 2 vs 3 here. Does that not work?